th2_check2_recon/services.py

```diff
@@ -252,6 +251,9 @@ def __init__(self, message_group_types: dict, cache_size: int, event_store: Even
         self.event_store = event_store
         self.rule_event = rule_event
 
+        if len(message_group_types.keys()) < 2:
```

Comment:
I suppose that checking `len` of dictionary rather than `len` of its keys is more Pythonic, but in terms of functionality it's the same.